tests/e2e/runner-pod/00-assert.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: kuttl.dev/v1beta1
```

Comment:
Suggestion: maybe have a readme in test directory explaining what the test does? It is non-trivial how this tests .Status.PodName

Reply:
This test is broader than just testing `.Status.PodName`. We test our basic guarantees around runner pod.

Reply:
Maybe add it as a separate commit then? For better blame.

But the suggestion still stands :laughing: